### ISSUE TITLE
☄️ Review Adventist's UI

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/adventist_knapsack#698]
# Summary

Adventist has gone through many updates as a result of upgrading their code to Valkyrie. Their UI will likely be impacted in a negative way. 

dependent on 
- [ ] #695 and
- [x] #696 

# Tasks

- [x] #745
- [ ] #744

# Notes

**Comment by ShanaLMoore** [comment]
![Image](https://github.com/user-attachments/assets/9bd46dc7-d3ef-4adc-9acb-0cdc38d346db)

**Comment by ShanaLMoore** [comment]
proprietor page: local vs. prod



![Image](https://github.com/user-attachments/assets/7306b8f4-7e26-45ed-a529-473bdf0b8cd0)

**Comment by ShanaLMoore** [comment]
Side bar: local vs. production

The local dashboard needs data for a fair comparison. 



![Image](https://github.com/user-attachments/assets/a62bd6ed-e498-40a1-8c95-aaf594b311ff)



#### collapse sidebar



![Image](https://github.com/user-attachments/assets/00a0ff4b-27be-4d8e-9113-b9fe5d2d4816)

**Comment by ShanaLMoore** [comment]
works index page: local vs. production

Note: Locally clicking the add new work button has an issue. You may be blocked on proceeding with works until that is resolved. 

![Image](https://github.com/user-attachments/assets/c0bd467a-0e7d-4139-bba2-b026bbfec84d)

**Comment by ShanaLMoore** [comment]
Click Add New Collection: local vs. production



![Image](https://github.com/user-attachments/assets/1ebc0d23-8555-4bd6-94dc-39ff6082ab0a)

**Comment by ShanaLMoore** [comment]
Click all collections tab: local vs production



![Image](https://github.com/user-attachments/assets/7e9b6feb-2d62-4348-9b7b-baa734c421f5)

**Comment by kirkkwang** [comment]
I recommend doing one big ticket to get all the obvious UI wonkiness out of the way and then from then on make separate tickets for anything else that was missed.

**Comment by ShanaLMoore** [comment]
@kirkkwang Do you have any helpful related PRs you can link to this?

**Comment by jillpe** [comment]
Issues being seen might be related to the bootstrap version

**Comment by kirkkwang** [comment]
@ShanaLMoore not really, i don't remember the dashboard ever being messed up.  Seems like we have some overrides going on because the dashboard should have looked fine.

**Comment by jillpe** [comment]
Blocked until more of the views are visible

**Comment by laritakr** [comment]
Work's show page breadcrumbs are incorrect... `main_app.polymorphic_path(presenter)` in `Hyrax::BreadcrumbsForWorks` returns nil, resulting in a link that goes nowhere. 

```
(byebug) main_app.polymorhphic_path presenter
*** NoMethodError Exception: undefined method `polymorhphic_path' for #<ActionDispatch::Routing::RoutesProxy:0x0000ffff617f8670 @scope=#<Hyrax::GenericWorksController:0x00000000108330>, @routes=#<ActionDispatch::Routing::RouteSet:0x0000ffff692d73a0>, @helpers=#<Module:0x0000ffff6927b410>, @script_namer=nil>
```

![Image](https://github.com/user-attachments/assets/7e30bdab-181f-45af-9e8c-5b9e9c4a1a8e)

**Comment by ShanaLMoore** [comment]
Closing. Client feedback should be captured in addressed in #744
